docs/adr/0020-use-internjs-testing.md

```diff
@@ -0,0 +1,69 @@
+# Use existing InternJS for functional testing in settings v2
+
+- Deciders: Vijay Budhram, Jody Heavener, Lauren Zugai
+- Date: 2020-07-17
+
+## Context and Problem Statement
+
+For over the past 5 years (at least), we have been using [InternJS](https://github.com/theintern/intern) for our functional tests in the content-server. While
+this has worked well and caught several bugs, we should reevaluate it since we are migrating to react and have an opportunity
+to use newer/faster/better supported testing frameworks.
+
+## Decision Drivers
+
+- Ease of writing tests
+- Good community support
+- Speed of tests
```

Comment:
I think we could add something here around the Settings learning curve - since we're already introducing new tech and conventions there, it makes sense to keep some things the same to mitigate that ramp up getting even steeper, as you note as a pro of keeping it. Maybe this just generally falls under "Ease of writing tests" 🤷